### PR TITLE
Feature/balanceOf148

### PR DESCRIPTION
**Pull request by AnnRiera**
This PR includes the `balanceOf` method for `HR721`.

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -0,0 +1,756 @@
+export const CONTRACT_ADDRESS = '0x16703afb468e4ba88380c2a2fda1aa4c5ec7b264'
```
Comment (bzuker) [inline src/constants.ts:1]: since this file is used only for testing, let's move it to a `/tests` folder

File: src/harmony-rpc.ts [hunk 2]
```diff
@@ -1,16 +1,18 @@
-import { getStatic } from '@ethersproject/properties'
+import { BigNumber } from '@ethersproject/bignumber'
```
Comment (bzuker) [inline src/harmony-rpc.ts:1]: let's change this and the spec file name to `HRC721`, the `base-hr721` to `base-HRC721`

File: src/harmony-rpc.ts [hunk 3]
```diff
@@ -21,21 +23,35 @@ export class BaseError extends Error {
...
     this.rpcProvider = provider
+    this.abi = abi
+    this.contract = new ethers.Contract(address, this.abi, this.rpcProvider)
   }
 
   async balanceOf(address: string): Promise<string> {
-    if (address) {
+    if (!address) {
       throw new Error('Balance query for the zero address')
     }
```
Comment (bzuker) [inline src/harmony-rpc.ts:38]: change the error msg to `You need to provide an address` or something like that

File: src/harmony-rpc.ts [hunk 4]
```diff
@@ -21,21 +23,35 @@ export class BaseError extends Error {
...
 
-    return ''
+    const balance = await this.contract.balanceOf(address)
```
Comment (bzuker) [inline src/harmony-rpc.ts:40]: move this inside the trycatch

**Review by bzuker — COMMENTED** [review]
some comments, but looking good

File: src/constants.ts (hunk 1 above)
Comment (AnnRiera) [reply src/constants.ts:1]: Okay, I'm coming!

File: src/harmony-rpc.ts (hunk 3 above)
Comment (AnnRiera) [reply src/harmony-rpc.ts:38]: Got it!

File: src/harmony-rpc.ts (hunk 4 above)
Comment (AnnRiera) [reply src/harmony-rpc.ts:40]: Roger that!

**Review by bzuker — APPROVED** [review]
one final comment but LGTM

File: src/HRC721.spec.ts [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  })
+
+  it('should be defined', () => {
+    expect(provider).to.not.be.undefined
+  })
+
+  describe('balanceOf', () => {
+    it('should get the number of tokens in the specified account', async () => {
+      const balance = await provider.balanceOf('0x36f41b8a79eca329610d6158f3ea9676bec281b9')
+      expect(balance).to.exist
```
[the remark this answers is not in the document]
Comment (AnnRiera) [reply src/HRC721.spec.ts:29]: Mmmh, you're right! No problem! I'll fix it

**Review by bzuker — APPROVED** [review]

**Pull request merged**